src/Knp/Menu/Util/MenuManipulator.php

```diff
@@ -208,7 +208,7 @@ public function toArray(ItemInterface $item, ?int $depth = null): array
      *
      * @param string|ItemInterface|array<int|string, mixed>|\Traversable<mixed>|null $subItem A string or array to append onto the end of the array
      *
-     * @phpstan-param string|ItemInterface|array<int|string, string|int|float|null|array{label: string, uri: string|null, item: ItemInterface|null}|ItemInterface>|\Traversable<string|int|float|null|array{label: string, uri: string|null, item: ItemInterface|null}|ItemInterface>|null $subItem
+     * @phpstan-param string|ItemInterface|array<int|string, string|int|float|null|array{label: string, uri: string|null, item: ItemInterface|null}|ItemInterface|object>|\Traversable<string|int|float|null|array{label: string, uri: string|null, item: ItemInterface|null}|ItemInterface|object>|null $subItem
```

Comment:
This must be reverted. the place where phpstan is reported an error is inside `testBreadcrumbsArrayInvalidData`, which is specifically testing the runtime behavior when passing invalid data (as not all users of the library might be using static analysis to catch such usage, and the code has some defense against that in that case).

the proper way to fix the phpstan warning is to use `@phpstan-expect-error` to make phpstan aware that an error is expected in that line.